themes/digital.gov/layouts/partials/core/footer.html

```diff
@@ -1,7 +1,7 @@
 <section class="return-to-top">
   <div class="grid-container grid-container-desktop usa-footer__return-to-top">
     <div class="grid-row">
-      <div role="navigation" class="grid-col-auto">
+      <div role="navigation" aria-label="Return to top" class="grid-col-auto">
```

Comment:
I would follow USWDS example in the [footer component ](https://designsystem.digital.gov/components/footer/).

It's inside of the footer and the markup is simplified

```html
<div class="grid-container usa-footer__return-to-top">
    <a href="#">Return to top</a>
</div>
```

Reply:
Thanks for the link, I've removed the `role=navigation`.